articles/dev-spaces/get-started-nodejs.md

```diff
@@ -62,7 +62,7 @@ Azure Dev Spaces isn't just about getting code running in Kubernetes - it's abou
 What happened? Edits to content files, like HTML and CSS, don't require the Node.js process to restart, so an active `azds up` command will automatically sync any modified content files directly into the running container in Azure, thereby providing a fast way to see your content edits.
 
 ### Test from a mobile device
-If you open the web app on a mobile device, you will notice that the UI does not display properly on a small device.
+Open the web app on a mobile device using the public URL for webfrontend. (You may want to copy and send the URL from your desktop to your device to save you from entering the long address.) When the web app loads in your mobile device, you will notice that the UI does not display properly on a small device.
```

Comment:
Remove `(` and `)` in the second sentence, they are not really needed.

Reply:
I removed the parentheses in last commit.

Reply:
Here as well.